scene/2d/area_2d.h

```diff
@@ -49,7 +49,7 @@ class Area2D : public CollisionObject2D {
 private:
 	SpaceOverride space_override = SPACE_OVERRIDE_DISABLED;
 	Vector2 gravity_vec;
-	real_t gravity = 0.0;
+	real_t gravity = 0;
```

Comment:
This change is unnecessary (especially considering that below there are still some `0.0`s).